code/modules/tooltip/tooltip.html

```diff
@@ -222,13 +231,13 @@
             mapHeight = map["view-size"].y,
             tilesShownX = tooltip.client_view_w;
           tilesShownY = tooltip.client_view_h;
-          (realIconSizeX = mapWidth / tilesShownX),
+          ((realIconSizeX = mapWidth / tilesShownX),
```

Comment:
```suggestion
          (realIconSizeX = mapWidth / tilesShownX),
```
I hate javascript because it is full of tiny subtle gotchas, and I know enough to know I don't think this should be meddled with without a real compelling reason. i do not know why vars are being declared like this in the orriginal code. i don't know why this even compiles. i feel sick.